internal/opts/parse.go

```diff
@@ -1,18 +1,18 @@
-/*Package filters provides tools for encoding a mapping of keys to a set of
-multiple values.
-*/
-package filters
+// Note that this file has originally been copied from github.com/docker/docker,
+// commit 86f080cff091 and been modified afterwards.
+//
+// Copyright 2012-2017 Docker, Inc.
+
+package opts
 
 import (
 	"encoding/json"
 	"errors"
 	"regexp"
 	"strings"
-
-	"github.com/docker/docker/api/types/versions"
 )
 
-// Args stores a mapping of keys to a set of multiple values.
+// args stores a mapping of keys to a set of multiple values.
```

Comment:
Nit: Why change case?

Reply:
Good catch. I was playing with the idea of making those types private but had to revert it at some point. I missed this comment.

Reply:
Fixed now.